src/config/section/network.js

```diff
@@ -285,7 +285,8 @@ export default {
           icon: 'plus',
           label: 'label.add.private.gateway',
           listView: true,
-          args: ['physicalnetworkid', 'vlan', 'ipaddress', 'gateway', 'netmask', 'sourcenatsupported', 'aclid'],
+          // TODO : Add version check
+          args: ['physicalnetworkid', 'vlan', 'bypassvlanoverlapcheck', 'ipaddress', 'gateway', 'netmask', 'sourcenatsupported', 'aclid'],
```

Comment:
@davidjumani no need, instead use the syntax to return args from a method (see example where args can be a `() => { return ...`). Basically you can use the example I showed you yesterday to check if param is available in the API to include in the args keys or no (i.e. check API discovery api.params if it exists, if yes return otherwise don't).